lua/jmod/cl_init.lua

```diff
@@ -502,6 +502,22 @@ net.Receive("JMod_NuclearBlast",function()
 	end
 end)
 
+activeBlindness = 0
+targetBlindness = 0
+oldBlindness = 0
+differenceBlindness = 0
```

Comment:
these can't be globals, i don't want them to be
make them values on the player, or put them all in a table or something on the player

Reply:
Ok, ironically I did that at first